eldritch/events.py

```diff
@@ -1718,6 +1722,12 @@ def resolve(self, state):
 
     encounters = [
         card.encounter_event(self.character, location_name) for card in self.draw.cards]
+    if any(isinstance(enc, Unimplemented) for enc in encounters):
```

Comment:
This should go above the section where we short-circuit if there is only one card drawn (the line that says TODO: test this). The short circuit only happens in test mode, so your two options are to turn test mode off to test this functionality or to move the if condition up.